spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -90,7 +90,7 @@
     context 'with admin user' do
       before { mock_user_sign_in(UserMock.new(:admin)) }
       context 'successful save' do
-        let!(:model) { FactoryGirl.build_stubbed(:equipment_model) }
+        let!(:model) { FactoryGirl.build(:equipment_model) }
```

Comment:
If it goes back, the test results in an error:
"stubbed models are not allowed to access the database --EquipmentModel#update_attribute(ordering,1)"